test/exercism/user_test.rb

```diff
@@ -18,8 +18,13 @@ def test_flipper_id_is_username
     assert_equal 'spocktocat', user.flipper_id
   end
 
+  def github_user(id, username, email, avatar_url)
+    GitHubUser.new("id" => id, "login" => username, "email" => email, "avatar_url" => avatar_url)
+  end
```

Comment:
I really don't like methods like this in tests that hide dependencies.
Could you use the `id: id, login: username` style?